wiki/Performance_points/id.md

```diff
@@ -0,0 +1,136 @@
+
+Sistem baru ini, yang kemudian dikenal sebagai "pp" (singkatan untuk "performance points"), diharapkan dapat mengubah patokan performa pemain sebelumnya yang sebenarnya hanya berasal dari total [skor](/wiki/Score) menjadi sesuatu yang benar-benar mencerminkan *skill*. Sistem baru ini disambut dengan baik oleh pemain pada saat itu.
+
+Beberapa bulan setelah pengungkapannya, osu! 20120722-24 secara resmi mengimplementasikan sistem tersebut untuk sepenuhnya mengganti sistem skoring [Ranked](/wiki/Beatmap/Category#ranked), dengan skor baru diperhitungkan setiap 30 menit. Pada Agustus di tahun yang sama, sistem ini ditingkatkan untuk meng-*update* secara *real-time*.
+
+*Catatan: ppv1, build awal dari sistem Performance Point, juga mempunyai sebuah changelog yang bisa dilihat di [topik forumnya](https://osu.ppy.sh/community/forums/topics/92185).*
+
+Sistem ini terus digunakan tanpa perubahan selama lebih dari satu tahun hingga [Tom94](https://osu.ppy.sh/users/1857058), pembuat dari pengitungan skor *osu!tp*, bergabung dengan [tim osu!](/wiki/People/The_Team) dan mengimplementasikan desainnya ke dalam sistem. Hasil dari sistem itu diberi nama *ppv2* dan dirilis pada tanggal 27 Januari 2014, yang membuat sistem lama itu diganti namanya menjadi *[ppv1](/wiki/Performance_points/ppv1)*
+
+Pada 16 Januari 2021, perubahan dilakukan pada sistem ppv2 yang bertujuan memberikan pp secara lebih akurat pada aspek yang lebih sulit pada suatu map. Perubahan ini sebagian besar dilakukan berkat bantuan berbagai pemain seperti [Xexxar](https://osu.ppy.sh/users/2773526) dan [StanR](https://osu.ppy.sh/users/7217455). Perubahan dijelaskan secara lebih detail di [post berita yang berhubungan](https://osu.ppy.sh/home/news/2021-01-14-performance-points-updates). Secara singkat, poin-poin perhatian utama pada pembaruan tersebut adalah sebagai berikut.
+
+- Mengenalkan faktor pengali untuk perolehan pp tambahan dari map dengan approach rate 11 berdasarkan durasi beatmap tersebut
+- Mengenalkan faktor pengali untuk pengurangan pp yang terpengaruhi oleh mod [No Fail](/wiki/Game_modifier/No_Fail) berdasarkan jumlah miss pada sebuah permainan
+- Mengenalkan faktor pengali untuk pengurangan pp yang terpengaruhi oleh mod [Spun Out](/wiki/Game_modifier/Spun_Out) berdasarkan jumlah spinner pada beatmap tersebut
+- Menyesuaikan nilai dari kerugian pp oleh miss untuk berdampak lebih kecil pada map lebih lama dengan kombo yang lebih tinggi
+- Mengurangi perolehan pp untuk [akurasi](/wiki/Accuracy) yang lebih rendah
+
+ppv2 pada saat ini bersifat aktif dengan pembaruan yang dirilis di [changelognya](https://osu.ppy.sh/p/changelog?category=pp).
+
+## Penghitungan
+
+Performance point memiliki tumpuan besar pada kalkulasi tingkat kesulitan suatu beatmap, di mana kalkulasi tersebut ditentukan oleh algoritma yang dibuat unik untuk masing-masing [game mode](/wiki/Game_mode).
```

Comment:
address the wiki to id once the new translate of game mode is merged

Reply:
just gonna do it now since it doesn't really make any differences